composer.json

```diff
@@ -21,6 +21,7 @@
         "cweagans/composer-patches": "^1.6.5",
         "dof-dss/nicsdru_origins_profile": "^0.1",
         "drupal-composer/drupal-scaffold": "^2.5",
+        "drupal/asset_injector": "^2.4",
```

Comment:
What issue does this module solve that isn't possible with theme assets? There's a potential security issue if this is misconfigured now, or in future, in the same manner as issues that can arise from text filters... so it seems sensible to question its presence.

Reply:
I guess it's allowing me to add styling to the admin theme without creating a sub theme or a custom module. If it's more appropriate I could try either of the other options.